cmd/operate/component.go

```diff
@@ -0,0 +1,102 @@
+package operate
+
+import (
+	"encoding/json"
+	"github.com/dream11/odin/internal/service"
+	fileUtil "github.com/dream11/odin/pkg/file"
+	"github.com/dream11/odin/pkg/util"
+	operate "github.com/dream11/odin/proto/gen/go/dream11/od/operate/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+var name string
+var serviceName string
+var env string
+var operation string
+var options string
+var file string
+var isComponent bool = true
+var componentClient = service.Component{}
+var config *structpb.Struct
+var operateComponentCmd = &cobra.Command{
+	Use:   "component",
+	Short: "operate component",
+	Args: func(cmd *cobra.Command, args []string) error {
+		return cobra.NoArgs(cmd, args)
+	},
+	Long: `odin operate component [Options]`,
+	Run: func(cmd *cobra.Command, args []string) {
+		execute(cmd)
+	},
+}
+
+func init() {
+	operateComponentCmd.Flags().StringVar(&name, "name", "", "name of the component")
+	operateComponentCmd.Flags().StringVar(&serviceName, "service", "", " name of the service in which the component is deployed")
+	operateComponentCmd.Flags().StringVar(&env, "env", "", "name of the environment in which the service is deployed")
+	operateComponentCmd.Flags().StringVar(&operation, "operation", "", "name of the operation to performed on the component")
+	operateComponentCmd.Flags().StringVar(&options, "options", "{}", "options of the operation in JSON format")
+	operateComponentCmd.Flags().StringVar(&file, "file", "", "path of the file which contains the options for the operation in JSON format")
+	if err := operateComponentCmd.MarkFlagRequired("name"); err != nil {
+		log.Fatal("Error:  --env, --name, --operation, --service cannot be blank", err)
```

Comment:
Change the error here. The error is when marking a flag as required is failing. see create/environment.go for an example error message for the same